pkg/logger/logger.go

```diff
@@ -164,3 +179,81 @@ func Named(l Logger, n string) Logger {
 	}
 	return ret
 }
+
+// Helper returns a logger 'skip' levels of callers skipped, if 'l' has a method `Helper(int) L`, where L implements Logger, otherwise it returns l.
+func Helper(l Logger, skip int) Logger {
```

Comment:
is there a more descriptive name for this? I would almost never expect Helper to be a public api.

is Helper zap lingo? What is a Helper method?

Reply:
Zap calls it 'caller skip' but it was confusing people so we settled on `Helper` a long time ago, since it's a helper for writing wrapping logger and utility functions :shrug: 
https://github.com/smartcontractkit/chainlink/blob/72ae3be34217d43c413b70c65d17a690994aef9e/core/logger/logger.go#L108-L110
In any case, we are just capturing and adapting the core logger here, and I would like to avoid exporting Helper from the relay logger.

Reply:
zap doesn't have a method for us to add to our interface. It requires calling `WithOptions`:
`l.SugaredLogger.WithOptions(zap.AddCallerSkip(skip))`

We could rename this func to something more intuitive, but I worry about breaking the symmetry.

Reply:
Added a link to the zap method at least.